python-template/{{cookiecutter.placeholder_repo_name}}/.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,39 +1,8 @@
-**Description:** Describe in a couple of sentences what this PR adds
-
-**JIRA:** Link to JIRA ticket
-
-**Dependencies:** dependencies on other outstanding PRs, issues, etc. 
-
-**Merge deadline:** List merge deadline (if any)
-
-**Installation instructions:** List any non-trivial installation 
-instructions.
-
-**Testing instructions:**
-
-1. Open page A
-2. Do thing B
-3. Expect C to happen
-4. If D happened instead - check failed.
-
-**Reviewers:**
-- [ ] tag reviewer 
-- [ ] tag reviewer 
 
 **Merge checklist:**
-- [ ] All reviewers approved
-- [ ] CI build is green
+Check off if complete or not applicable:
 - [ ] Version bumped
 - [ ] Changelog record added
 - [ ] Documentation updated (not only docstrings)
-- [ ] Commits are squashed
-
-**Post merge:**
-- [ ] Create a tag
-- [ ] Check new version is pushed to PyPI after tag-triggered build is 
-      finished.
-- [ ] Delete working branch (if not needed anymore)
-
-**Author concerns:** List any concerns about this PR - inelegant 
-solutions, hacks, quick-and-dirty implementations, concerns about 
-migrations, etc.
+- [ ] Commits are squashed into one or more [conventional commits](https://open-edx-proposals.readthedocs.io/en/latest/best-practices/oep-0051-bp-conventional-commits.html)
+- [ ] Noted any: Concerns, dependencies, migration issues, deadlines, tickets, testing instructions
```

Comment:
```suggestion
- [ ] Noted any: Concerns, dependencies, migration issues, deadlines, tickets
- Either:
  - [ ] Manually tested, with steps and/or screenshots provided
  - [ ] Manual testing is not necessary
```

At risk of bloating the list: *Did you test this, and if so, how?* is probably the number-one question I ask in PR reviews. I love that OpenCraft's PR template includes it as an item and I think it makes reviewing their PRs much easier. I suggest adapting it here.

Reply:
I think those are not mutually exclusive. If a reviewer adds a user-facing feature, I'd want them to both add unit tests and check that it works manually.

Then, there are some changes where automatic testing isn't possible (eg CSS changes) and some changes where testing just isn't required (eg reformatting a file).